tinkergraph-gremlin/src/main/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerGraph.java

```diff
@@ -340,6 +342,14 @@ private void validateHomogenousIds(final List<Object> ids) {
         }
     }
 
+    @Override
+    public TinkerGraph clone() {
+        TinkerGraph cloned = new TinkerGraph(configuration);
```

Comment:
please `final` variables as we tend to do that as a style throughout the code base